java/api/src/main/java/io/ray/api/Ray.java

```diff
@@ -318,48 +321,62 @@ public static void exitActor() {
   /**
    * Get a placement group by placement group Id.
    *
+   * @deprecated This method is no longer recommended to create a new placement group, use {@link
```

Comment:
remove "to create a new placement group"

Reply:
updated.